### PR TITLE
services/horizon: Build and test verify-range Docker image

### PR DESCRIPTION
**Pull request by 2opremio**
Part of #2761

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -383,6 +383,18 @@ jobs:
...
+    docker:
+      - image: docker:18.04-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build and test Verify Range Docker image
+          command: |
+            docker build -f services/horizon/docker/verify-range/Dockerfile -t stellar/horizon-verify-range services/horizon/docker/verify-range/
+            docker run -ti -e BRANCH=master -e FROM=15000063 -e TO=15000127 stellar/horizon-verify-range
```
Comment (2opremio) [inline .circleci/config.yml:396]: Maybe there is a better range to test, but I didn't want it to take more than a few minutes.

@bartekn let me know if you have other range suggestions

Comment (paulbellamy) [reply .circleci/config.yml:396]: If there's other ranges to test, we could do a few different ranges in parallel on circle?

Comment (paulbellamy) [reply .circleci/config.yml:396]: Also, it would be nice to comment in the circle file why we chose these particular ranges.

**Review by paulbellamy — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (bartekn) [reply .circleci/config.yml:396]: I think it's fine to have a single range here. Isn't it added here just to check if the image works? I have some ideas when it comes to ranges but the image is run using the code from `master` so it wouldn't really test the PR.

**Review by bartekn — APPROVED** [review]

**Comment by 2opremio** [comment]
CodeQL seems stuck, I will merge without it.

**Comment by 2opremio** [comment]
Uhm, I rebased and the verify-range test gets stuck, I suspect it could be due to #3576

I will remove the PR and see if it passes

**Comment by 2opremio** [comment]
I think it is due to #3576  (it works with `BRANCH=05e09089828a0528e5545a04d7f713dee8f8e379`). @paulbellamy could you take a look?

**Comment by 2opremio** [comment]
OK, it seems to _randomly_ get stuck (it just worked with master).

**Comment by 2opremio** [comment]
I am thinking it may be a memory problem, so I am going to move to an even earlier range.

**Pull request merged**